public/js/lib/chrome-remote-debug-protocol/bootstrap.js

```diff
@@ -42,6 +42,7 @@ InspectorBackend.registerCommand("Page.screencastFrameAck", [{"name": "sessionId
 InspectorBackend.registerCommand("Page.handleJavaScriptDialog", [{"name": "accept", "type": "boolean", "optional": false}, {"name": "promptText", "type": "string", "optional": true}], [], false);
 InspectorBackend.registerCommand("Page.setColorPickerEnabled", [{"name": "enabled", "type": "boolean", "optional": false}], [], false);
 InspectorBackend.registerCommand("Page.setOverlayMessage", [{"name": "message", "type": "string", "optional": true}], [], false);
+InspectorBackend.registerCommand("Page.configureOverlay", [{ "name": "suspended", "type": "boolean", "optional": true },{"name": "message", "type": "string", "optional": true}], [], false);
```

Comment:
Just added this in by hand to get it working with Canary without removing the function above it.